util/util.go

```diff
@@ -29,6 +32,24 @@ func GenerateUUID() string {
 	return uuid.New().String()
 }
 
+
+func Transport(pURL string) *http.Transport {
```

Comment:
It would be better if you rename it to something like `ProxyTransport` or `TransportProxy`

Reply:
The README has the details.